pkg/scheduler/scheduler.go

```diff
@@ -350,6 +352,8 @@ func (sched *Scheduler) bindVolumesWorker() {
 			Status: v1.ConditionFalse,
 			Reason: reason,
 		})
+
+		metrics.BindPodVolumeLatency.Observe(metrics.SinceInMicroseconds(start))
```

Comment:
It may also be good to add a metric to count VolumeBindingFailed.  I don't expect it to happen often.

Reply:
@msau42 Done